pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -19,9 +19,11 @@ in
 , buildTarget ? ""
 , buildTools ? [], libraryToolDepends ? [], executableToolDepends ? [], testToolDepends ? [], benchmarkToolDepends ? []
 , configureFlags ? []
+, buildFlags ? []
 , description ? ""
 , doCheck ? !isCross && (stdenv.lib.versionOlder "7.4" ghc.version)
 , doBenchmark ? false
+, doVerbose ? false
```

Comment:
In the presence of `buildFlags` this attribute seems redundant?

Reply:
I agree that we can drop this in favor or passing `-v3` to the `buildFlags`.